lamp/tasks/install_php.yml

```diff
@@ -1,32 +1,44 @@
 ---
+- set_fact:
+    php_version: 7.0
+  when: php_version is not defined
+
+- name: add php repo
+  become: yes
+  apt_repository:
+    repo: 'ppa:ondrej/php'
+    state: present
+    update_cache: yes
+  when: php_version != 7.0
+
 - name: install php
   become: yes
   apt:
-      name: "{{ item }}"
+      name: "php{{ '' if php_version == '7.0' else php_version }}-{{ item }}"
```

Comment:
Did you find a particular item that was not available as `php7.0-foo`? I did a quick search for a few (fpm, curl, common), and they were installable as, for example, `php7.0-fpm`.